src/main/java/org/betonquest/betonquest/api/Variable.java

```diff
@@ -13,6 +14,13 @@ abstract public class Variable {
      */
     protected Instruction instruction;
 
+    /**
+     * A variable marked as static can be executed even if the playerID in the
+     * {@link #getValue(String)} method is null
+     */
+    @Getter
+    protected boolean staticness;
+
```

Comment:
If you use the `@Getter` you also need to write valid JavaDocs for it. It is a bit complicated, but you can see an valid example in BetonQuest.java line 295 https://github.com/BetonQuest/BetonQuest/blob/b110da3da5912a24a05c92d177409ea4c1fbf9cc/src/main/java/org/betonquest/betonquest/BetonQuest.java#L288